Dropdown/tests/Dropdown-specs.js

```diff
@@ -1,261 +1,271 @@
 		);
 
-		const expected = false;
-		const actual = dropDown.find('[aria-labelledby]').exists();
+		const actual = screen.getByRole('region');
 
-		expect(actual).toBe(expected);
+		expect(actual).not.toHaveAttribute('aria-labelledby');
 	});
 
 	test('should have `placeholder` when `children` is invalid', () => {
-		const dropDown = shallow(
+		render(
 			<DropdownBase placeholder={placeholder}>
 				{null}
 			</DropdownBase>
 		);
 
-		const expected = placeholder;
-		const actual = dropDown.find('DropdownButton').prop('children');
+		const actual = screen.getByText(placeholder);
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeInTheDocument();
 	});
 
 	test('should have `placeholder` that reflects `selected` option', () => {
 		const selectedIndex = 1;
-
-		const dropDown = shallow(
+		render(
 			<DropdownBase selected={selectedIndex}>
 				{children}
 			</DropdownBase>
 		);
 
-		const expected = children[selectedIndex];
-		const actual = dropDown.find('DropdownButton').prop('children');
+		const actual = screen.getByText(children[selectedIndex]);
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeInTheDocument();
 	});
 
 	test('should have `placeholder` when `selected` is invalid', () => {
-		const dropDown = shallow(
+		render(
 			<DropdownBase placeholder={placeholder} selected={-1}>
 				{children}
 			</DropdownBase>
 		);
 
-		const expected = placeholder;
-		const actual = dropDown.find('DropdownButton').prop('children');
+		const actual = screen.getByText(placeholder);
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeInTheDocument();
 	});
 
 	test('should be disabled when `children` is omitted', () => {
-		const dropDown = mount(
+		render(
 			<DropdownBase title={title} />
 		);
 
-		const expected = true;
-		const actual = dropDown.find('DropdownButton').prop('disabled');
+		const expected = 'true';
+		const actual = screen.getByRole('button');
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveAttribute('aria-disabled', expected);
 	});
 
 	test('should be disabled when there are no `children`', () => {
-		const dropDown = mount(
+		render(
 			<DropdownBase title={title}>
 				{[]}
 			</DropdownBase>
 		);
 
-		const expected = true;
-		const actual = dropDown.find('DropdownButton').prop('disabled');
+		const expected = 'true';
+		const actual = screen.getByRole('button');
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveAttribute('aria-disabled', expected);
 	});
 
 	test('should update when children are added', () => {
-		const dropDown = shallow(
+		const {rerender} = render(
 			<Dropdown title={title}>
 				{children}
 			</Dropdown>
 		);
 
 		const updatedChildren = children.concat('option4', 'option5');
-		dropDown.setProps({children: updatedChildren});
+
+		rerender(
+			<Dropdown title={title}>
+				{updatedChildren}
+			</Dropdown>
+		);
 
 		const expected = 5;
-		const actual = dropDown.children().length;
+		const actual = updatedChildren.length;
```

Comment:
Unfortunately, this does not verify that the actual dropdown has 5 options, but just that an array is updated. You will proably need to change the test to render an opened dropdown and check the number of children of a node(something like screen.getByRole("list").children.length

Reply:
You are right. Thank you for the suggestion!